seatunnel-dist/release-docs/LICENSE

```diff
@@ -220,16 +220,20 @@ The text of each license is the standard Apache 2.0 license.
      (APL2) msgpack-core (org.msgpack:msgpack-core:0.9.0 - https://msgpack.org/)
      (Apache 2) Apache Derby Client JDBC Driver (org.apache.derby:derbyclient:10.14.2.0 - http://db.apache.org/derby/)
      (Apache 2) Apache Derby Database Engine and Embedded JDBC Driver (org.apache.derby:derby:10.12.1.1 - http://db.apache.org/derby/derby/)
+     (Apache 2) Apache Derby Database Engine and Embedded JDBC Driver (org.apache.derby:derby:10.14.2.0 - http://db.apache.org/derby/)
      (Apache 2) Apache Derby Network Server (org.apache.derby:derbynet:10.14.2.0 - http://db.apache.org/derby/)
      (Apache 2) JDO API (javax.jdo:jdo-api:3.0.1 - http://db.apache.org/jdo)
      (Apache 2) JavaEWAH (com.googlecode.javaewah:JavaEWAH:0.3.2 - http://code.google.com/p/javaewah/)
      (Apache 2) Joda time (joda-time:joda-time:1.6 - http://joda-time.sourceforge.net)
+     (Apache 2) Joda-Time (joda-time:joda-time:2.9.3 - http://www.joda.org/joda-time/)
+     (Apache 2) Joda-Time (joda-time:joda-time:2.9.9 - http://www.joda.org/joda-time/)
```

Comment:
Can you explain why so many jars are newly introduced?

Reply:
You mean licenses?

Reply:
yup

Reply:
You can refer to https://github.com/apache/incubator-seatunnel/pull/1372#discussion_r818375324

Reply:
I'm sure most people don't want to heavily modify this file over and over again. License is very important, it is best to list the differences before merging with #1210.

Reply:
I can agree with your idea. Because this situation is not usual, it would be few change in future like this one https://github.com/apache/incubator-seatunnel/pull/1372

Reply:
I can’t agree with your idea. Because this situation is not usual, it would be few change in future like this one https://github.com/apache/incubator-seatunnel/pull/1372

Reply:
If the user needs to add a new jar, does he also need to follow such a sorting rule?

Reply:
The script can help to add new licenses and put them in the right position, so users wouldn’t need to do it manually, even this sort rule is by character, which is simply.

Reply:
I think we have already communicated on this issue. What we need is only to print what needs to be added in the log, rather than requiring users to learn Python-related things and then execute Python scripts. You have to know that the main developers of ST are still Java-based.

Reply:
Yes it is printing log, I mean it also can be able to add

Reply:
It just copy-paste thing, so it is not hard like you said

Reply:
Usually adding a jar contains multiple dependencies, we are to simplify the contributor, not to increase the threshold for him. Increasing the collation is not a good idea, for me it's cumbersome.

Reply:
I mean, no sort of rule is needed. that's it

Reply:
The aim of script is reduced the threshold of users

Reply:
> It just copy-paste thing, so it is not hard like you said

Try not to let contributors add it manually, so this script is useless

Reply:
What do you mean issues..

Reply:
It is clear enough I believe 😂

Reply:
What do you mean issue…